lib/clickhouse-activerecord/migration.rb

```diff
@@ -20,7 +20,7 @@ def create_table
           distributed_suffix = "_#{full_config[:distributed_service_tables_suffix] || 'distributed'}"
         end
 
-        connection.create_table(table_name.concat(distributed_suffix.to_s), **table_options) do |t|
+        connection.create_table(table_name + distributed_suffix.to_s, **table_options) do |t|
```

Comment:
was getting `FrozenError: can't modify frozen String: "schema_migrations"` in our CI pipeline when running `bundle exec rails db:schema:load RAILS_ENV=test`